delta/app/src/main/scala/ch/epfl/bluebrain/nexus/delta/routes/ProjectsRoutes.scala

```diff
@@ -44,9 +51,35 @@ final class ProjectsRoutes(identities: Identities, acls: Acls, projects: Project
       }
     }
 
+  private def authorizeForProjectUUID(orgUuid: UUID, projectUuid: UUID, permission: Permission)(implicit
+      caller: Caller
+  ): Directive1[ProjectResource] =
+    onSuccess(projects.fetch(orgUuid, projectUuid).runToFuture).flatMap {
+      case Some(project) =>
+        authorizeFor(AclAddress.Project(project.value.organizationLabel, project.value.label), permission).tmap(_ =>
```

Comment:
We could have an apply method that create a `AclAddress.Project` from a project ref ?
So we would have here:
`AclAddress.Project(project.value.ref)`